spec.bs

```diff
@@ -2444,6 +2448,28 @@ algorithms to achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-notification.https.html
 </wpt>
 
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The [[!CSSOM-VIEW]] specification defines the {{Element/scrollIntoView()}} method that calls the
+[=scroll a target into view=] algorithm. This will not only scroll the {{Element}} or
+[=css2/viewport=] to make the target visible, but will also scroll [=tree/ancestor=]s if necessary
+to make the target visible, essentially causing the scroll to "bubble up". This means that
+{{Element/scrollIntoView()}} performed in a [=child navigable=] or [=fenced navigable container/
+fenced navigable=] can be observed by its embedder, allowing for collusion across a fenced frame
+boundary. This section patches the [=scroll a target into view=] algorithm to prevent that collusion
+at the expense of some utility.
+
+<div algorithm=scroll-target-into-view>
+  Modify the [=scroll a target into view=] algorithm to add a step at the end of the algorithm that
+  reads:
+
+  14. If <var ignore>scrolling box</var>'s associated {{Element}} or [=css2/viewport=]'s associated
+     {{Document}}'s [=node navigable=]'s [=navigable/traversable navigable=] is a [=fenced navigable
+     container/fenced navigable=], terminate these steps.
```

Comment:
"Terminate these steps" is a little unclear just because technically any single instance of this algorithm is indeed terminating after the end of these steps, because the recursion is implicit in the definition and not explicit in the spec steps. So I think we need to say something more aggressive here to combat the implicit recursion allowed for in the definition of this algorithm. Maybe ", let this be the last instance of this algorithm that stops any further recursive instances that would otherwise follow."